apis/mlcoreapi/mlcoreapi.yaml

```diff
@@ -448,6 +448,12 @@ paths:
           schema:
             type: string
           example: 2515e6fbe5039a6db09c10148e3c743b
+        - name: createProject
+          in: query
+          description: Specify project need to be created or not.
```

Comment:
Please check if the description can be modified as "Specify whether the project needs to be created or not."